website/docs/advanced/override_grammar/basic.md

```diff
@@ -173,6 +186,13 @@ As an example, in the following `dir` is set to the string `job{a=1,b=2,c=3}`:
 $ python my_app.py 'dir=job\{a\=1\,b\=2\,c\=3\}'
 ```
 
+Some characters, such as the bar symbol `'|'`, are not legal in unquoted strings.
+Strings containing such characters must be quoted:
+
+```shell
+$ python my_app.py 'dir="A|B"'
+```
+
```

Comment:
Since "|" is also special in shell, it would be good to distinguish to what "|" is not legal.

Reply:
PR #1850 is merged, so Hydra's parser now accepts the pipe character in unquoted strings.
I've updated the above to use the left-bracket character `'['` and have added some language to disambiguate that Hydra's parser is rejecting the character (as opposed to the shell rejecting it).